src/hypothesis/_settings.py

```diff
@@ -373,17 +373,6 @@ class Setting(object):
 )
 
 settings.define_setting(
-    'max_mutations',
```

Comment:
This removal is not mentioned in change log.

Reply:
That's from #535 - because this setting was never documented, it's not public API and we can therefore remove it without a deprecation period.

Reply:
Take my words back here.